src/ems_service.ts

```diff
@@ -50,10 +48,6 @@ export abstract class AbstractEmsService implements IEmsService {
     return attributions.join('|');
   }
 
-  getOrigin(): string {
```

Comment:
`getOrigin` is still used in Kibana [here](https://github.com/elastic/kibana/blob/6693ef371f887eca639b09c4c9b15701b4ebabd4/src/plugins/maps_ems/public/service_settings/service_settings.ts#L88) and [here](https://github.com/elastic/kibana/blob/6693ef371f887eca639b09c4c9b15701b4ebabd4/src/plugins/maps_ems/public/service_settings/service_settings.ts#L132). Are we expecting to remove usages there as well?

Reply:
yes, we're removing this here https://github.com/elastic/kibana/pull/104422

Reply:
Ah, OK, I missed that.